integration/ct_integration_test.go

```diff
@@ -43,34 +47,63 @@ func TestCTIntegration(t *testing.T) {
 	fmt.Printf("Today's test has been brought to you by the letters C and T and the number %#x\n", *seed)
 	rand.Seed(*seed)
 
-	cfg, err := ctfe.LogConfigFromFile(*logConfigFlag)
+	cfgs, err := ct.LogConfigFromFile(*logConfigFlag)
 	if err != nil {
 		t.Fatalf("Failed to read log config: %v", err)
 	}
 
+	if err := runParallelTests(cfgs, *httpServersFlag, *testDir, *mmdFlag); err != nil {
```

Comment:
Not sure I follow how to do that -- `t.Parallel` appears to run different tests in parallel, whereas this is running multiple instances of the same test in parallel.

(BTW, the Travis builds include a variant with `-race` set, so we can hopefully catch concurrency issues.)